shell/platform/windows/window_win32.cc

```diff
@@ -527,6 +529,17 @@ void WindowWin32::Destroy() {
 }
 
 void WindowWin32::HandleResize(UINT width, UINT height) {
+  bool next_mimimized = (width <= 0) && (height <= 0);
+  if (current_mimimized != next_mimimized) {
+    current_mimimized = next_mimimized;
+
+    if (next_mimimized) {
+      OnMinimized();
+    } else {
+      OnRestoredFromMinimized();
+    }
+  }
```

Comment:
"Mimimized" should be "minimized" 😄 

```suggestion
  bool next_minimized = (width <= 0) && (height <= 0);
  if (current_minimized_ != next_minimized) {
    current_minimized_ = next_minimized;

    if (next_minimized) {
      OnMinimized();
    } else {
      OnRestoredFromMinimized();
    }
  }
```

Reply:
done